src/commands/redis_cmd.cc

```diff
@@ -154,6 +160,9 @@ Status ParseTTL(const std::vector<std::string> &args, std::unordered_map<std::st
       }
     }
   }
+  if (px_exist + ex_exist + exat_exist + pxat_exist >= 2) {
```

Comment:
```suggestion
  if (px_exist + ex_exist + exat_exist + pxat_exist > 1) {
```

nit: More than 1 means has duplicates. Also, it's my personal bias. Not a blocker from merging.